components/observations/SimpleForm.tsx

```diff
@@ -175,7 +195,37 @@ export const SimpleForm: React.FC<{
     },
     [images, setImages],
   );
-  const renderOverlay = useCallback((index: number) => <ImageListOverlay index={index} onPress={removeImage} />, [removeImage]);
+
+  const editImage = useCallback(
+    (index: number) => {
+      const image = images[index];
+      if (image == null) {
```

Comment:
nit: here, elsewhere - is a `== null` check more brittle than truthiness? Are we just inferring that `image` can't be `undefined`?

Reply:
Using `== null` is a standard JS way of refining out `T | null | undefined` to `T`. There's even a lint for it to prevent you from doing just a truthiness check: https://typescript-eslint.io/rules/strict-boolean-expressions/

And for refining out `null` explicitly truthiness checks have weird edge cases https://eslint.org/docs/latest/rules/no-eq-null

As far as using truthiness, there's enough weird edge cases around `if (image) {` that there's also a lint for no doing truthiness checks unless `image` is strictly a `boolean.` 

Index access of a Record type is also considered unsafe and there's stricter flags to make that the case:
https://www.typescriptlang.org/tsconfig#noUncheckedIndexedAccess

These are all things I have turned on in my projects so this pattern is somewhat muscle memory for me.

And JavaScript is ridiculous.

Reply:
TL;DR in this case a truthiness check is fine so I can change, just muscle memory for those specific lints.

Reply:
Thanks as always for the deep insights! If we don't have those stricter flags on in this project, please feel free to turn them on! JavaScript is certainly ridiculous, thanks for helping me learn!